knowledge-base/tabstrip-add-remove-tabs.md

```diff
@@ -0,0 +1,171 @@
+---
+title: Add and Remove Tabs
+description: Learn how to dynamically add and remove tabs
+type: how-to
+page_title: Add and Remove Tabs
+slug: tabstrip-kb-add-remove-tabs
+tags: telerik,blazor,tabstrip,add tabs,remove tabs
+ticketid: 
+res_type: kb
+previous_url: /knowledge-base/tabstrip-dynamic-tabs
+---
+
+## Environment
+
+<table>
+    <tbody>
+        <tr>
+            <td>Product</td>
+            <td>TabStrip for Blazor</td>
+        </tr>
+    </tbody>
+</table>
+
+
+## Description
+
+I have a collection of items representing separate tabs. I am iterating through that collection to render a tab for each item as shown [here](slug:tabstrip-tabs-collection). I want to allow the user to add and remove tabs, how to achieve that?
+
+This KB article answers the following questions:
+
+* How to remove a tab with a "X" button in the tab header?
```

Comment:
Consider rephrasing this. Do you mean that the user wants to _remove the tab by using the X button_, or do you mean that the _user wants to remove a tab that has an X button in the header_?